setup-ceph.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd /ceph
+git clone https://github.com/openattic/ceph .
+./install-deps.sh
+./do_cmake.sh $@
+cd /ceph/build
+make -j$(nproc)
```

Comment:
How am I suppose to set this "nproc", is this missing in the README?

Reply:
This is  a shell builtin, no need to set it manually.

Reply:
All those steps described in the REAMDE aren't necessary as it'll suffice to call `setup-ceph` from within the Docker container.
From what I understood how rebuilding Ceph works, it should also be enough to remove the `-e` from the `setup-ceph.sh` file to be able to call it again, once you want to rebuild Ceph.